src/loaders/compute_tools/eggnog/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM continuumio/miniconda3:24.1.2-0
+
+ENV EGGNOG_VER 2.1.12
+ENV CONDA_ENV eggnog-$EGGNOG_VER
+ENV PYTHON_VER 3.11
+
+RUN conda config --add channels bioconda
+RUN conda config --add channels conda-forge
+
+RUN conda create -n $CONDA_ENV python=$PYTHON_VER
+RUN conda install -n $CONDA_ENV -c conda-forge -c bioconda eggnog-mapper=$EGGNOG_VER
+RUN conda install -n $CONDA_ENV pandas=2.2.1 jsonlines=2.0.0
+
+RUN echo "source activate $CONDA_ENV" >> ~/.bashrc
+
+# eggNOG annotation DB is pre-downloaded at /global/cfs/cdirs/kbase/collections/libraries/eggnog/5.0.2
```

Comment:
5.0.2 is the DB version?

Reply:
yea. That's the eggNOG DB version. There are also diamond DB and MMseqs DB. But I use the eggNOG DB version.
```
(eggnog-2.1.12) root@919d2e0de08e:/app# emapper.py --version
emapper-2.1.12 / Expected eggNOG DB version: 5.0.2 / Installed eggNOG DB version: 5.0.2 / Diamond version found: diamond version 2.1.9 / MMseqs2 version found: 15.6f452 / Compatible novel families DB version: 1.0.1
```

Reply:
And the instructions you reference explain how to install all those dbs and they're all in the same dir?

Reply:
yea. The I followed the instructions to install the DB. They are all in the same dir by the download_eggnog_data.py program comes with eggnog installation.

Reply:
:+1: